bin/mycroft-config

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 Mycroft AI Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SOURCE="${BASH_SOURCE[0]}"
+cd -P "$( dirname "$SOURCE" )"
+DIR="$( pwd )"
+script=${0}
+script=${script##*/}
+
+function help() {
+    echo "${script}:  Mycroft configuration manager"
+    echo "usage: ${script} [COMMAND] [params]"
+    echo
+    echo "COMMANDs:"
+    echo "  edit (system|user)                  edit and validate config file"
+    echo "  reload                              instruct services to reload config"
+    echo "  show (default|remote|system|user)   display the specified setting file"
+    echo "  set <var>                           set the variable (under USER)"
+    echo "  get [var]                           display a particular variable"
+    echo "                                      or all if no 'var' specified"
+    echo "Note: Use jq format for specifying <var>"
+    echo
+    echo "Examples:"
+    echo "  ${script} edit user"
+    echo "  sudo ${script} edit system"
+    echo "  ${script} show remote"
+    echo "  ${script} get"
+    echo "  ${script} get enclosure.platform"
+    echo "  ${script} set test.subvalue \"foo\" "
+
+    exit 1
+}
+
+################################################################
+# Setup stuff based on the environment
+
+VIEWER="nano --syntax=json --view"
+if [ -z "$EDITOR" ] ; then
```

Comment:
should the sensible-editor link be checked as an alternative?

Reply:
Done